lib/server.ml

```diff
@@ -107,7 +107,7 @@ let connect channel ?offer ?(no_tls=false) () =
       end
     in loop ()
   in
-  let rec negotiate_tls () =
+  let negotiate_tls make_tls_channel = let rec negotiate_tls () =
```

Comment:
I think I'd find this more readable in a new line. Two `let`s in one line is too much for me to process :smile: